airbyte-integrations/connectors/source-snowflake/src/main/java/io.airbyte.integrations.source.snowflake/SnowflakeSource.java

```diff
@@ -113,4 +107,10 @@ private JsonNode buildUsernamePasswordConfig(final JsonNode config, final String
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  @Override
```

Comment:
How come this method was moved since it seems like nothing was changed? Correct me if I'm wrong

Reply:
Sorry, but these are not my changes. I don't know why GitHub displays them
![2022-08-26_15-02](https://user-images.githubusercontent.com/88329385/186899291-83c6e316-f118-469a-9035-a2f64dbe2c26.png)
.